spring-boot/springboot-sti/springboot-sti-all.json

```diff
@@ -33,7 +33,7 @@
             "source": {
                "type": "Git",
                "git": {
-                  "uri": "https://github.com/jorgemoralespou/osev3-examples.git",
+                  "uri": "https://github.com/iceman91176/osev3-examples.git",
```

Comment:
I can't merge this back into my repo :-D
I'm fine with the other changes, though.